compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -50,9 +50,15 @@ static llvm::cl::opt<int> clNativeVectorSizeInBytes(
 
 static llvm::cl::opt<int> clNumberOfRuntimeThreads(
     "iree-codegen-llvm-number-of-threads",
-    llvm::cl::desc("number of threads that are used at runtime"),
+    llvm::cl::desc("number of threads that are used at runtime if codegen "
+                   "thread distribution is enabled"),
     llvm::cl::init(8));
 
+static llvm::cl::opt<bool> clEnableDistribution(
+    "iree-codegen-llvm-enable-distribution",
```

Comment:
Nit: I'd rather make this `disable-distribution`. We do distribution by default. So if you want to disable it you need to do something.